architecture/service_catalog/aws_broker.adoc

```diff
@@ -0,0 +1,70 @@
+[[admin-guide-get-started-aws]]
+= {product-title} Amazon Web Services Broker
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
```

Comment:
We should highlight how this is supported! 
While redhat will help with installs and troubleshooting, the primary support for this is handle by AWS.

Reply:
@sferich888  Something like:
Red Hat provides support for the installation of the AWS Broker and troubleshooting of the OpenShift cluster and Service Catalog issues, primary support for the AWS Broker is handled by Amazon Web Services." 
?